src/gui/editors/SongEditor.cpp

```diff
@@ -1031,25 +1012,18 @@ SongEditorWindow::SongEditorWindow(Song* song) :
 
 	// Set slider zoom
 	m_zoomingSlider = new AutomatableSlider(m_toolBar, tr("Zoom"));
-	m_zoomingSlider->setValue(m_editor->m_zoomingLinearModel->value());
-	m_zoomingSlider->setModel(m_editor->m_zoomingLinearModel);
+	m_zoomingSlider->setModel(m_editor->m_zoomingModel);
 	m_zoomingSlider->setOrientation(Qt::Horizontal);
 	m_zoomingSlider->setPageStep(1);
+	m_zoomingSlider->setFocusPolicy(Qt::NoFocus);
```

Comment:
The focused widgets receive scroll events first. If they ignore the event, it will be sent to the widget under the cursor. So if the slider gets focus by clicking it you won't be able to ctrl+scroll.